src/lambda_function.py

```diff
@@ -1,20 +1,19 @@
 import time
 
 from webdriver_wrapper import WebDriverWrapper
+from selenium.webdriver.common.keys import Keys
 
 
 def lambda_handler(*args, **kwargs):
     driver = WebDriverWrapper()
 
     driver.get_url('https://www.google.es/')
 
-    driver.set_input_value('//input[@id="lst-ib"]', '21 buttons')
-
-    driver.click('//center//img[@alt="Google"]')
-    time.sleep(0.5)
+    driver.set_input_value('//input[@name="q"]', '21 buttons')
 
+    button = driver.find("//input[@name='btnK']")
+    button.send_keys(Keys.TAB)
```

Comment:
Why you do this?

Reply:
So we will focus that element, without this I got `ElementNotVisibleException` constantly.